packages/nextjs/src/server/errors.ts

```diff
@@ -29,10 +29,20 @@ Alternatively, you can set your own ignoredRoutes. See https://clerk.com/docs/ne
 `;
 
 export const getAuthAuthHeaderMissing = () =>
-  'You need to use "authMiddleware" (or the deprecated "withClerkMiddleware") in your Next.js middleware file. You also need to make sure that your middleware matcher is configured correctly and matches this route or page. See https://clerk.com/docs/quickstarts/get-started-with-nextjs';
+  `Clerk: auth() was called but it looks like you aren't using authMiddleware in your middleware file. Please ensure the following:
+- Use "authMiddleware" (or the deprecated "withClerkMiddleware").
```

Comment:
It's time to remove the deprecated proposal completely.

Reply:
Done